django/contrib/admin/static/admin/css/rtl.css

```diff
@@ -148,6 +148,11 @@ form .aligned .checkbox-row + .help {
     padding-right: 0;
 }
 
+form .aligned p.datetime div.help.timezonewarning {
```

Comment:
You might be able to add it into the above, something like this:
```css
form .aligned .checkbox-row + .help,
form .aligned p.datetime div.help.timezonewarning {
    margin-right: 0;
    padding-right: 0;
}
```

Reply:
Good point, merged.